File: test/Runner.cs
using System.Globalization;

public class Runner
{
	public static int Main()
	{
		CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
		if (Test.Run()) {
			System.Console.WriteLine("PASSED");
			return 0;
		}
		else {
			System.Console.WriteLine("FAILED");
			return 1;
		}
	}
}

File: test/Runner.java
import java.util.Locale;

public class Runner
{
	public static void main(String[] args)
	{
		Locale.setDefault(Locale.ROOT);
		if (Test.run()) {
			System.out.println("PASSED");
		}
		else {
			System.out.println("FAILED");
			System.exit(1);
		}
	}
}
